corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -36,10 +33,9 @@
   <script>
     $(function() {
 
-      var loc_id = {{ location.get_id|JSON }};
       var hierarchy = {{ hierarchy|JSON }};
       model = new LocationSelectViewModel(hierarchy, '\u2026', false, function(loc) {
```

Comment:
I had to look it up - looks like a unicode character corresponding to an ellipsis.  The function parameter it corresponds to is called `default_caption`.